Documentation/intro-to-cli.md

```diff
@@ -47,7 +47,41 @@ You can get a sense of using the tools from the examples below.
 Design
 ======
 
-More content here.
+There are couple of moving pieces that you make up the general design of the .NET Core CLI:
```

Comment:
Should this be "There are *a* couple of moving pieces *that* make up the general design of the .NET Core CLI:"?